src/classes/Bucket.ts

```diff
@@ -273,7 +273,7 @@ export class Bucket {
   };
 
   /**
-   * allows lender to kick a loan based on a LUP calculated as if they withdraw liquidity
+   * Allows lender to kick a loan based on a LUP calculated as if they withdraw liquidity.
```

Comment:
nit: `Kick` a loan with a lender's liquidity based on a LUP calculated as if they withdraw liquidity

Reply:
Adressed in 4dc9c72f655776daa1b9a36f70526e836760235f